test/core/event_engine/thread_pool_test.cc

```diff
@@ -221,6 +221,22 @@ TEST_F(WorkStealingThreadPoolTest,
   p.Quiesce();
 }
 
+// TODO(hork): This is currently a pathological case for the original thread
+// pool, it takes around 50s to run. When that is fixed, or the implementation
+// is deleted, make this a typed test again.
+TEST_F(WorkStealingThreadPoolTest, QuiesceRaceStressTest) {
```

Comment:
Why does this take a long time to run ?

Reply:
I haven't profiled it in earnest since that code is going away, but based on a rough cycle count, the original pool will more often (non-deterministically) take longer in either quiesce or destruction. Generally, the old code is more likely to get stuck in a few spots.

```
D0601 17:31:31.160825129      12 thread_pool_test.cc:180]              DO NOT SUBMIT: run_time:496.500000  , quiesce_time:96.500000  , destroy_time:1001072.000000
D0601 17:31:32.161774568      12 thread_pool_test.cc:180]              DO NOT SUBMIT: run_time:640.750000  , quiesce_time:1000272.250000  , destroy_time:626.000000
D0601 17:31:32.162480260      12 thread_pool_test.cc:180]              DO NOT SUBMIT: run_time:450.500000  , quiesce_time:214.000000  , destroy_time:1000957.750000
D0601 17:31:32.163147749      12 thread_pool_test.cc:180]              DO NOT SUBMIT: run_time:443.750000  , quiesce_time:185.750000  , destroy_time:699.500000
```